yew/src/virtual_dom/vlist.rs

```diff
@@ -34,6 +41,20 @@ impl DerefMut for VList {
     }
 }
 
+#[cfg(feature = "ssr")]
+impl TryFrom<VList> for Html {
+    type Error = HtmlStringifyError;
+
+    fn try_from(value: VList) -> Result<Html, HtmlStringifyError> {
+        let mut parts: Vec<String> = vec![];
```

Comment:
Have you considered using a mutable string instead of a vector to build these?
I don't really have a strong opinion on this but I feel like using a string would be clearer.
There's the [`push_str`](https://doc.rust-lang.org/std/string/struct.String.html#method.push_str) method for adding a string and [`push`](https://doc.rust-lang.org/std/string/struct.String.html#method.push) for individual characters.

Using a string instead of a vector should also be (significantly) more performant.
Here's a benchmark of various string concatenation methods: https://github.com/hoodie/concatenation_benchmarks-rs. It's really old though so I'm not sure if the results still hold up.
```
test mut_string_push_str           ... bench:          75 ns/iter (+/- 35) <- String::push_str
test mut_string_push_string        ... bench:         164 ns/iter (+/- 77) <- Vec::<String>::join
```